rootfs/usr/local/bin/run.sh

```diff
@@ -15,6 +15,15 @@ if [ "$PHP_HARDENING" == "true" ] && [ ! -f /usr/local/etc/php/conf.d/snuffleupa
     cp /usr/local/etc/php/snuffleupagus/* /usr/local/etc/php/conf.d
 fi
 
+# Check if database is available
+if [ ${DB_TYPE} != "sqlite3" ]; then
```

Comment:
This check does not work for people who only define their database in `config.php` and not as an environment variable.
One solution would be to check also whether a DB_TYPE is set at all, e.g.:
```bash
if [ -n "${DB_TYPE}" ] && [ "${DB_TYPE}" != "sqlite3" ]; then
```

Reply:
I updated the commit accordinly.